src/modules/ChannelSettings/components/ModerationPanel/MembersModal.tsx

```diff
@@ -65,10 +68,11 @@ export function MembersModal({
             },
           })}
         >
-          {members.map((member: Member) => (
+          {members.map((member: Member, index) => (
             <React.Fragment key={member.userId}>
               {
                 renderUserListItem({
+                  index,
```

Comment:
@HoonBaek Any reason we pass index here? It seems there's no usage of this index.

Reply:
I thought this will be useful when customer wants to add the current user item in Modals
Because in Modal, we can't divide only user list